projects/fsastorefrontlib/src/core/my-account/facade/quote.service.ts

```diff
@@ -220,6 +214,24 @@ export class QuoteService {
       .unsubscribe();
   }
 
+  loadQuotesComparison(quoteCodes: string[]) {
+    return this.userIdService.getUserId().pipe(
+      take(1),
+      map(occUserId => {
+        this.store.dispatch(
+          new fromAction.LoadQuoteComparison({
+            cartCodes: quoteCodes,
+            userId: occUserId,
+          })
+        );
+      })
+    );
+  }
+
+  getQuotesComparison() {
+    return this.store.pipe(select(fromQuoteStore.getQuotesComparison));
```

Comment:
Maybe add `take(1)` after select just to be on the safe side.

Reply:
There is no need for this, whenever the store's state is changed, it emits data only ones. With `take(1)` ruins everything, try and see.